src/main/webapp/app/assessment-instructions/expandable-problem-statement/expandable-problem-statement.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ProgrammingExercise } from 'app/entities/programming-exercise';
+import { SafeHtml } from '@angular/platform-browser';
+import { ArtemisMarkdown } from 'app/components/util/markdown.service';
+import { Exercise, ExerciseType } from 'app/entities/exercise';
+
+@Component({
+    selector: 'jhi-expandable-problem-statement',
+    templateUrl: './expandable-problem-statement.component.html',
+    styleUrls: ['../assessment-instructions.scss'],
+})
+export class ExpandableProblemStatementComponent {
+    @Input() exercise: ProgrammingExercise;
+    @Input() isCollapsed = false;
```

Comment:
This is not passed from the outside, so I guess it should not be marked as `@Input`